src/book/03-guides/03-hardware/02-servo-calibration.mdx

```diff
@@ -0,0 +1,102 @@
+
+5. Rotate the servo so that the dot is in the 6-o'clock position (as seen in the instructions). Use the tool pictured to move the horn and use the marks on the tool to line up the dot correctly.
+
+![Photo of the servo rotating tool](./images/servo_calibration/servo_tool.jpg)
+
+6. Continue to rotate to each of the 3-o'clock, 12-o'clock and 9-o'clock positions when told by the software.
+
+## Parameters
+
+To update the servo parameters, perform the following instructions:
+
+1. Select "**Scan**" and wait for the attached servo to be discovered.
+
+   Optional: brand new servos should be preset to an ID of 1. Lower the ID range in "**Options > Scan**" to speed up discovery.
+   ![Screenshot of Dynamixel Wizard performing a scan with no servos discovered](./images/servo_calibration/scan.png)
+   ![Screenshot of Dynamixel Wizard performing a scan with 3 servos discovered](./images/servo_calibration/discover.png)
+
+2. If there are multiple servos discovered, the new servo should be the option with ID = 1 and Baud Rate = 57600 bps.
+
+3. Update the following parameters (don't forget to press save after each update):
+
+   - ID: Varies depending on where the servo will be placed. See below for guidance.
+
+     | Servo Position       | ID  |
+     | -------------------- | --- |
+     | Right Shoulder Pitch | 1   |
+     | Left Shoulder Pitch  | 2   |
+     | Right Shoulder Roll  | 3   |
+     | Left Shoulder Roll   | 4   |
+     | Right Elbow          | 5   |
+     | Left Elbow           | 6   |
+     | Right Hip Yaw        | 7   |
+     | Left Hip Yaw         | 8   |
+     | Right Hip Roll       | 9   |
+     | Left Hip Roll        | 10  |
+     | Right Hip Pitch      | 11  |
+     | Left Hip Pitch       | 12  |
+     | Right Knee           | 13  |
+     | Left Knee            | 14  |
+     | Right Ankle Pitch    | 15  |
+     | Left Ankle Pitch     | 16  |
+     | Right Ankle Roll     | 17  |
+     | Left Ankle Roll      | 18  |
+     | Head Yaw             | 19  |
+     | Head Pitch           | 20  |
+
+     Note that roll is about the x-axis, pitch is about the y-axis, and yaw is about the z-axis.
+
+   - Baud Rate (Bus): 1M bps = 2M/(1 + 1)
+
+     ![Screenshot of Dynamixel Wizard Baud Rate selection with 1M bps selected](./images/servo_calibration/update_parameters.png)
+
+## Testing
+
+To test the servo, perform the following instructions:
+
+1. Set the Goal Position in Dynamixel Wizard to 0, +90 and -90 and observe the position of the arrow on the horn. It should match the picture.
+
+2. Set the Goal Position in Dynamixel Wizard to some position either side of +180 to test if the servo is in joint mode. If it is it should take the long route around to the second position. If it does not, it is not in joint mode. _instructions on how to fix_
```

Comment:
I don't understand `_instructions on how to fix_` ? is that meant to be a  link? A header for the next section perhaps?

Reply:
Probably a "insert instructions here" placeholder.